src/components/CardContainer/FilmCard/FilmCard.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { Card, Grid, CardMedia, CardActionArea, Checkbox } from "@mui/material";
+import { FavoriteBorder, Favorite } from "@mui/icons-material";
+import { CardInfoSection } from "./CardInfoSection/CardInfoSection";
+import { RatingLabel } from "./RaitingLabel/RaitingLabel";
```

Comment:
import from folder root as you have added index.ts

Reply:
done